conf/janus.eventhandler.mqttevh.jcfg.sample

```diff
@@ -16,16 +16,21 @@ general: {
 							# plain (no indentation) or compact (no indentation and no spaces)
 
 	url = "tcp://localhost:1883"	# The URL of the MQTT server. Only tcp supported at this time.
-	client_id = "janus.example.com"	# Janus client id. You have to configure a unique ID.
+	client_id = "janus.example.com"	# Janus client id. You have to configure a unique ID (default: guest).
 	#keep_alive_interval = 20		# Keep connection for N seconds (default: 30)
 	#cleansession = 0				# Clean session flag (default: off)
-	#mqtt_qos = 1					# Default MQTT QoS for published events
+	#retain = 0						# Default MQTT retain flag for published events
+	#qos = 1						# Default MQTT QoS for published events
```

Comment:
Why have you changed the name of that property? That's a breaking change for people using the plugin in production right now.

Reply:
Apologies, my mistake... just noticed that it's the name in the config that's wrong, and that it's actually "qos" in the code :+1:

Reply:
Yep, I just adjusted config properties with real property names in code.

Reply:
Thanks for catching those mistakes! (which I should have catched myself in the first place)